ngraph/test/onnx/onnx_import.in.cpp

```diff
@@ -2713,6 +2713,71 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_model_scatterND_const_i32_indices)
     test_case.run();
 }
 
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_gather_elements_float_1D)
+{
+    const auto function = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/gather_elements_float_1D.prototxt"));
+    auto test_case = test::TestCase<TestEngine>(function);
+
+    test_case.add_input<float>({1, 2, 3});
+    test_case.add_input<int64_t>({1});
+    test_case.add_expected_output<float>(Shape{1}, {2});
+
+    test_case.run();
+}
+
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_gather_elements_int8_axis_1)
+{
+    const auto function = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/gather_elements_int8_axis_1.prototxt"));
+    auto test_case = test::TestCase<TestEngine>(function);
+
+    test_case.add_input<int8_t>({1, 2, 3, 4});
+    test_case.add_input<int32_t>({0, 0, 1, 0});
+    test_case.add_expected_output<int8_t>(Shape{2, 2}, {1, 1, 4, 3});
+
+    test_case.run();
+}
+
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_gather_elements_int32_axis_0)
+{
+    const auto function = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/gather_elements_int32_axis_0.prototxt"));
+    auto test_case = test::TestCase<TestEngine>(function);
+
+    test_case.add_input<int32_t>({1, 2, 3, 4, 5, 6, 7, 8, 9});
+    test_case.add_input<int64_t>({1, 2, 0, 2, 0, 0});
+    test_case.add_expected_output<int32_t>(Shape{2, 3}, {4, 8, 3, 7, 2, 3});
+
+    test_case.run();
+}
+
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_gather_elements_float_negative_axis)
+{
+    const auto function = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/gather_elements_float_negative_axis.prototxt"));
+    auto test_case = test::TestCase<TestEngine>(function);
+
+    test_case.add_input<float>({1, 2, 3, 4});
+    test_case.add_input<int64_t>({1, 1, 1, 0});
+    test_case.add_expected_output<float>(Shape{2, 2}, {2, 2, 4, 3});
+
+    test_case.run();
+}
+
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_gather_elements_float_3D_axis_2)
+{
+    const auto function = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/gather_elements_float_3D_axis_2.prototxt"));
+    auto test_case = test::TestCase<TestEngine>(function);
+
+    test_case.add_input<float>({1, 2, 3, 4, 5, 6, 7, 8});
+    test_case.add_input<int64_t>({0, 1, 0, 1});
+    test_case.add_expected_output<float>(Shape{2, 2, 1}, {1, 4, 5, 8});
+
+    test_case.run();
+}
```

Comment:
It's nice to always verify the behavior with dynamic shapes, I don't know if it is necessary in this case.